backend/.env.example

```
@@ -12,4 +12,19 @@ JWT_SECRET=your_super_secret_jwt_key_here_replace_with_secure_random_string
-DEFAULT_BUSINESS_ID=default_business_123
\ No newline at end of file
+DEFAULT_BUSINESS_ID=default_business_123
+
+# Email Configuration (SMTP)
+# Required for email notifications feature
+SMTP_HOST=smtp.gmail.com
+SMTP_PORT=587
+SMTP_USER=your_email@gmail.com
+SMTP_PASSWORD=your_app_password
+SMTP_FROM_EMAIL=your_email@gmail.com
+SMTP_FROM_NAME=Inventory Billing System
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SMTP_FROM_NAME key should go before the SMTP_HOST key

(UnorderedKey)